.travis.yml

```diff
@@ -18,6 +18,7 @@ before_script:
   - composer install --prefer-source
   - if [ "$DEPENDENCIES" != "low" ]; then composer update; fi;
   - if [ "$DEPENDENCIES" == "low" ]; then composer update --prefer-lowest; fi;
+  - if [[ $DB == "mysql" || $DB == "mariadb" ]]; then SCHEMA=`xmllint --xpath 'string(//var[@name="db_name"]/@value)' tests/travis/$DB.travis.xml` && mysql -e "CREATE SCHEMA $SCHEMA; GRANT ALL PRIVILEGES ON $SCHEMA.* to travis@'%'"; fi;
```

Comment:
Surely not just for this line. We can actually hardcode it in `.travis.yml` though...